protocol/triple/triple_protocol/codec.go

```diff
@@ -19,6 +19,10 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	hessian "github.com/apache/dubbo-go-hessian2"
+	"github.com/dubbogo/grpc-go/encoding"
+	"github.com/dubbogo/grpc-go/encoding/proto_wrapper_api"
```

Comment:
How about we make a copy of the definitions we needed in `dubbogo/grpc-go`? I think it would be better to not depend on  `dubbogo/grpc-go` except for some compatibility needs.

Reply:
I would use new protoc-gen-go-triple to generate new code.